src/components/Header.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Text,
+  View,
+  StyleSheet,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+
+import CustomIcon from './CustomIcon';
+import {
+  BORDERRADIUS,
+  COLORS,
+  FONTFAMILY,
+  FONTSIZE,
+  SPACING,
+} from '../theme/theme';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingTop: SPACING.space_36,
+    paddingHorizontal: SPACING.space_36,
+  },
+  back_icon: {
+    backgroundColor: COLORS.Orange,
+    padding: 7,
+    borderRadius: BORDERRADIUS.radius_25,
+    width: 34,
+    height: 34,
+  },
+  header_text: {
+    color: COLORS.White,
```

Comment:
prefer to keep the variable names under the constants as lowercase, here i.e. `White`

Reply:
will do this in final PR ...so that i can change it everywhere i used